### ISSUE TITLE
NoMethodError: undefined method `colorize' for #<String:0x007fa23c16e708>

### ISSUE THREAD

**Issue by nahankid** [mattbrictson/capistrano-mb#14]
cap production mb:crontab
(Backtrace restricted to imported tasks)
cap aborted!
NoMethodError: undefined method `colorize' for #<String:0x007fa23c16e708>

Tasks: TOP => mb:crontab
(See full trace by running task with --trace)

**Comment by nahankid** [comment]
Happens during compatibility warning:

```
            if File.exist?(override_path)
              compatibility_warning(
                "Please move #{override_path} from lib/capistrano/fiftyfive "\
                "to lib/capistrano/mb to ensure future compatibility with "\
                "capistrano-mb."
              )
            end
```

Changed the path from fiftyfive to mb and it works. So the severity is very low.

**Comment by mattbrictson** [comment]
Thank you for the bug report! I'll fix this in an upcoming release.

**Comment by mattbrictson** [comment]
Fixed via 68c01a5
